servers/rendering_server.cpp

```diff
@@ -2874,6 +2874,10 @@ void RenderingServer::init() {
 	GLOBAL_DEF("rendering/rendering_device/staging_buffer/texture_upload_region_size_px", 64);
 	GLOBAL_DEF("rendering/rendering_device/descriptor_pools/max_descriptors_per_pool", 64);
 
+	// Number of commands that can be drawn per frame.
+	GLOBAL_DEF_RST("rendering/gl_compatibility/item_buffer_size", 16384);
+	ProjectSettings::get_singleton()->set_custom_property_info("rendering/gl_compatibility/item_buffer_size", PropertyInfo(Variant::INT, "rendering/gl_compatibility/item_buffer_size", PROPERTY_HINT_RANGE, "1024,1048576,1"));
```

Comment:
Does it need to be a power of two or is that just a good habit? If it's a requirement, this could maybe be exposed as an enum to ensure users don't mistakenly use something unoptimal like 20000.

Reply:
It's just a nice round number. It doesn't need to be a power of two.